### PR TITLE
126 aggregate test results

### LINKED ISSUE #126: Aggregate results of all tests
Add aggregation of test results to all process that spawn several subprocesses

### PR DESCRIPTION
**Pull request by pihme**
closes #126 

This adds a new worker to aggregate test results. This worker is also added to workflows which spawn several tests/sub-workflows.

### CODE REVIEW

File: workflows/run-all-tests-in-camunda-cloud.bpmn [hunk 1]
```diff
@@ -65,32 +65,84 @@
...
+      <bpmn:incoming>Flow_03g3kjv</bpmn:incoming>
+      <bpmn:outgoing>Flow_03cdsbb</bpmn:outgoing>
+      <bpmn:startEvent id="Event_1qw2vfw">
+        <bpmn:outgoing>Flow_0908wsg</bpmn:outgoing>
+      </bpmn:startEvent>
+      <bpmn:endEvent id="Event_1p56h3p">
+        <bpmn:incoming>Flow_0908wsg</bpmn:incoming>
+      </bpmn:endEvent>
+      <bpmn:sequenceFlow id="Flow_0908wsg" sourceRef="Event_1qw2vfw" targetRef="Event_1p56h3p" />
+    </bpmn:subProcess>
```
Comment (deepthidevaki) [inline workflows/run-all-tests-in-camunda-cloud.bpmn:88]: Do we need a subprocess for this? Can we just set the default value before itself?

**Review by deepthidevaki — APPROVED** [review]
LGTM :+1:

File: workflows/run-all-tests-in-camunda-cloud.bpmn (hunk 1 above)
Comment (pihme) [reply workflows/run-all-tests-in-camunda-cloud.bpmn]: Brilliant. This might just work.

Cannot believe I missed this simple solution, so thanks for that!

**Pull request merged**